contracts/StandardBounty.sol

```diff
@@ -0,0 +1,319 @@
+pragma solidity ^0.4.18;
+import "./inherited/StandardToken.sol";
+
+/// @title StandardBounties
+/// @dev Used to pay out individuals or groups for task fulfillment
+
+contract StandardBounty {
+
+  /*
+   * Storage
+   */
+
+  address public masterCopy;
+
+  address public controller;
+  string public data;
+
+  bool public hasPaidOut;
+
+  Fulfillment[] public fulfillments;
+
+  Contribution[] public contributions;
+
+
+
+  /*
+   * Events
+   */
+  event BountyFulfilled(uint256 _fulfillmentId);
```

Comment:
In gerneral will just make a blanket statement here. We need to pass more through these events.